cmd/oci/oci.go

```diff
@@ -61,6 +62,10 @@ var OCICmd = &cobra.Command{
 		}
 		platform = p
 
+		if tag != "" {
+			artifact.Name = fmt.Sprintf("%s:%s", artifact.Name, tag)
```

Comment:
if `artifact.Name` contains a `:` (ex: a user passed `ttl.sh/app:1h` and they want to override the tag with `3h`), we should allow that.